### PR TITLE
region expansion P1

### PR DESCRIPTION
**Pull request by micheleRP**
## Description

Resolves https://github.com/redpanda-data/documentation-private/issues/2606 
Review deadline: Monday, July 29

## Page previews
[What's New in Redpanda Cloud](https://deploy-preview-636--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/whats-new-cloud/)
[BYOC regions](https://deploy-preview-636--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/byoc-tiers/#byoc-supported-regions)
[Dedicated regions](https://deploy-preview-636--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/dedicated/dedicated-tiers/#dedicated-supported-regions)

## Checks

- [ ] New feature
- [ ] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

**Comment by micheleRP** [comment]
@towfiqa I added also in the What's New. I did not include eu-central-1 (Frankfurt) from the doc issue in that list, because it was already in our docs.

**Review by Feediver1 — APPROVED** [review]
lgtm

File: modules/deploy/pages/deployment-option/cloud/whats-new-cloud.adoc [hunk 1]
```diff
@@ -5,6 +5,15 @@
 
 This page lists new features added in Redpanda Cloud.
 
+== July 2024
+
+=== Support for additional regions
+
+For xref:deploy:deployment-option/cloud/byoc-tiers.adoc#byoc-supported-regions[BYOC clusters] and xref:deploy:deployment-option/cloud/dedicated/dedicated-tiers.adoc#dedicated-supported-regions[Dedicated clusters], Redpanda added support for the following regions:
+
+* GPC: asia-east1 (UAE), asia-northeast1 (Tokyo), southamerica-east1 (São Paulo)
```
Comment (towfiqa) [inline modules/deploy/pages/deployment-option/cloud/whats-new-cloud.adoc]: This should be GCP not GPC

File: modules/deploy/pages/deployment-option/cloud/whats-new-cloud.adoc [hunk 2]
```diff
@@ -5,6 +5,15 @@
...
+* AWS: ap-east-1 (Hong Kong), ap-northeast-1 (Tokyo), me-central-1 (UAE)
```
Comment (towfiqa) [inline modules/deploy/pages/deployment-option/cloud/whats-new-cloud.adoc:28]: This is missing Frankfurt as per the Aha https://redpanda.aha.io/features/CD-173

File: modules/deploy/pages/deployment-option/cloud/whats-new-cloud.adoc (hunk 1 above)
Comment (towfiqa) [inline modules/deploy/pages/deployment-option/cloud/whats-new-cloud.adoc]: This incorrectly says UAE. This should be Taiwan as per the Aha https://redpanda.aha.io/features/CD-173

**Review by towfiqa — CHANGES_REQUESTED** [review]
Please make changes and propagate through to the other pages

File: modules/deploy/pages/deployment-option/cloud/whats-new-cloud.adoc (hunk 2 above)
Comment (micheleRP) [reply modules/deploy/pages/deployment-option/cloud/whats-new-cloud.adoc:28]: @towfiqa Frankfurt was already documented, that's why I didn't add it to the What's New

**Review by towfiqa — APPROVED** [review]

**Comment by micheleRP** [comment]
removed AWS me-central-1 in https://github.com/redpanda-data/docs/pull/656

**Pull request merged**